src/index.html

```diff
@@ -11,7 +11,150 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
```

Comment:
fix your attributes formatting, [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) 3rd item
```suggestion
    <form 
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
    >
```